coalib/results/Diff.py

```diff
@@ -286,9 +286,9 @@ def unified_diff(self):
         for equality comparison.
         """
         return ''.join(difflib.unified_diff(
-            self.original,
-            self.modified,
-            tofile=self.rename if isinstance(self.rename, str) else ''))
+                self.original,
+                self.modified,
+                tofile=self.rename if isinstance(self.rename, str) else ''))
```

Comment:
is this indentation necessary for this commit? -> New commit.
As it's a codestyle change only, rather omit this change completely as the previous version is also valid.